static/app/views/detectors/components/detailsPanel.tsx

```diff
@@ -0,0 +1,45 @@
+import styled from '@emotion/styled';
+
+import {Flex} from 'sentry/components/container/flex';
+import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+
+// TODO: Make component flexible for different alert types
+function DetailsPanel() {
+  return (
+    <Panel column gap={space(0.5)} justify="flex-start">
+      <strong>{t('Query:')}</strong>
+      <Query>
+        <Subtext>{t('visualize:')}</Subtext> <p>{t('p75')}</p>
+        <Subtext>{t('where:')}</Subtext> <p>{t('level is error')}</p>
+        <Subtext>{t('grouped by:')}</Subtext> <p>{t('release')}</p>
+      </Query>
+      <strong>{t('Threshold:')}</strong>
+    </Panel>
+  );
+}
+
+const Panel = styled(Flex)`
+  background-color: ${p => p.theme.backgroundSecondary};
+  border: 1px solid ${p => p.theme.translucentBorder};
+  border-radius: ${p => p.theme.borderRadius};
+  padding: ${space(1.5)};
+`;
```

Comment:
Is this worth making into it's own component? We'll need this kind of gray box for automations detail as well

Reply:
If these are different than the existing `Panel` components, the yes. Maybe a new one called `components/workflowEngine/ui/container`?